src/aggregators/bracketing.jl

```diff
@@ -35,23 +38,23 @@ BracketData{T1, T2}() where {T1, T2} = BracketData(T1(0.1), T2(25), T2(4))
     end
 end
 
+@inline get_spec_brackets(bd, i, u::AbstractVector) = get_spec_brackets(bd, i, u[i])
+
+"Get propensity brackets of massaction jump k."
 @inline function get_majump_brackets(ulow, uhigh, k, majumps)
     evalrxrate(ulow, k, majumps), evalrxrate(uhigh, k, majumps)
 end
 
 # for constant rate jumps we must check the ordering of the bracket values
+"Get propensity brackets of constant rate jump."
 @inline function get_cjump_brackets(ulow, uhigh, rate, params, t)
     rlow = rate(ulow, params, t)
     rhigh = rate(uhigh, params, t)
-    if rlow <= rhigh
-        return rlow, rhigh
-    else
-        return rhigh, rlow
-    end
+    return min(rlow, rhigh), max(rlow, rhigh)
```

Comment:
```suggestion
    return (rlow <= rhigh) ? rlow,rhigh : rhigh,rlow
```
if you want it as a one-liner.